notebooks/04_model_select_and_optimize/hypertune_spark_deep_dive.ipynb

```diff
@@ -166,7 +166,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
    "source": [
```

Comment:
Better say "runs [hyperparameter tuning"  rather than "is supposed to run hyperparameter tuning".

Reply:
"It is worth mentioning that tuning hyperparameter with `hyperdrive` in Azure Machine Learning services will be a feature in the future version.". This contradicts the previous paragraph, which says how to do tuning with AML.

Reply:
👍